pkg/ccl/cliccl/load_test.go

```diff
@@ -207,3 +195,24 @@ func checkExpectedOutput(t *testing.T, expected string, out string) {
 	output := out[endOfCmd+1:]
 	require.Equal(t, expected, output)
 }
+
+// Generate n continuous TimeStamps with minimal interval of 10 milliseconds to be used in incremental backup tests.
+func generateContinuousTimeStamps(n int) []hlc.Timestamp {
+	timeStamps := make([]hlc.Timestamp, 0, n)
+	if n == 0 {
+		return timeStamps
+	}
+	lastTS := hlc.Timestamp{WallTime: timeutil.Now().UnixNano()}
```

Comment:
nit: this may be a bit easier to read if `lastTS` is a `time.Time` and it's only converted to an hlc.Timestamp when added to `timeStamps`.